manifest.yml

```diff
@@ -0,0 +1,9 @@
+name: netlify-plugin-sitemap
+inputs:
+  - name: baseUrl
+    description: Site's home URL. Defaults to Netlify's SITE URL.
```

Comment:
running this branch locally, the `description` field was causing problems. [according to the documentation](https://github.com/netlify/build#validating-plugin-inputs)

```
The inputs property is an array of objects with the following members:

    name {string}: Name of the input. Required.
    required {boolean}
    default {any}: Default value.
```

Reply:
Good point!